### DISCUSSION TITLE
Remove/Rephrase description for config:best-practices

### DISCUSSION THREAD

**Discussion by jvmdc** [renovatebot/renovate#31500] [Suggest an Idea]
### Tell us more.

When onboarding a repository while extending from the `config:best-practices`, Renovate will create this description:

> Based on the default config's presets, Renovate will:
> 
>   - Start dependency updates only once this onboarding PR is merged
>   - Enable Renovate Dependency Dashboard creation.
>   - Use semantic commit type `fix` for dependencies and `chore` for all others if semantic commits are in use.
>   - Ignore `node_modules`, `bower_components`, `vendor` and various test/tests directories.
>   - Group known monorepo packages together.
>   - Use curated list of recommended non-monorepo package groupings.
>   - Apply crowd-sourced package replacement rules.
>   - Apply crowd-sourced workarounds for known problems with packages.
>   - Pin Docker digests.
>   - Pin `github-action` digests.
>   - Pin dependency versions for `devDependencies`.
>   - Preset with best practices from the Renovate maintainers. Recommended for advanced users, who want to follow our best practices.

The last sentence, coming from the description of `config:best-practices` itself, does not make sense in the context: "Renovate will: Preset with best practices from the Renovate maintainers. Recommended for advanced users, who want to follow our best practices.`

I propose that the description is either changed to something that makes sense in the context of this list, removed entirely or made possible to avoid having the description of a preset being added to the list?

(We build on top of `config:best-practices` and a few other presets, but deliberately avoid having a description in our "preset-of-presets" since it simply uses other presets - Each underlying preset will already have added what's worth to add to the list))

Also huge thanks for this piece of software, it makes a lot of chores a lot easier!

**Comment by rarkins** [comment]
Fixed by https://github.com/renovatebot/renovate/pull/31517

Note: PR body is only updated after a new commit is done to the onboarding branch, which usually only happens if the default branch has a commit.

**Comment by jvmdc** [comment]
Great, thank you!

Description not being output when a preset only consists of presets makes sense - Is this behaviour documented anywhere? It's nice to know when creating our own bundles of presets

**Reply by rarkins** [reply]
No, but you're welcome to submit a PR adding that if you find a good place for it
